containers/runner/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN dnf -y update && \
     rsync \
     git \
     virt-install \
+    libvirt-daemon \
```

Comment:
Not sure i follow but sure this is needed to run VMs.

Reply:
It contains libvirtd, in F38 it was pulled in by some other virt package I guess, in F39 I think there was some packages consolidation in libvirt.